llama_index/evaluation/tonic_validate/answer_consistency.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any, Optional, Sequence
+
+from llama_index.evaluation.base import BaseEvaluator, EvaluationResult
+from llama_index.prompts.mixin import PromptDictType, PromptMixinType
+
+
+class AnswerConsistencyEvaluator(BaseEvaluator):
+    """Tonic Validate's answer consistency metric.
+
+    See https://docs.tonic.ai/validate/ for more details.
+
+    Args:
+        openai_service(OpenAIService): The OpenAI service to use. Specifies the chat
+            completion model to use as the LLM evaluator. Defaults to "gpt-4".
+    """
+
+    def __init__(self, openai_service: Optional[Any] = None):
+        import_err_msg = (
+            "`tonic-validate` package not found, please run `pip install "
+            "tonic-validate`"
+        )
+        try:
+            from tonic_validate.metrics.answer_consistency_metric import (
+                AnswerConsistencyMetric,
+            )
+            from tonic_validate.services.openai_service import OpenAIService
+        except ImportError:
+            raise ImportError(import_err_msg)
+
+        if openai_service is None:
+            openai_service = OpenAIService("gpt-4")
```

Comment:
is it possible to use a different llm-judge?

Reply:
We currently support the family of gpt-4 and gpt-3.5 OpenAI models as llm-judges.